TriggertrapSLR/SunriseSunsetCalculatorViewController.swift

```diff
@@ -549,7 +549,7 @@ class SunriseSunsetCalculatorViewController: SplitLayoutViewController, CLLocati
     fileprivate func updateSunConstraint() {
         let diagramCalculations = self.whiteView.frame.size.width / 2 - self.sunriseDiagramLabel.frame.origin.x - self.sunriseDiagramLabel.frame.size.width
         let sunviewCalculations = self.sunImageView.frame.size.width / 2.0 - self.sunRotationCenterView.frame.size.width / 2.0
-        self.sunToCenterConstraint.constant =  diagramCalculations - sunviewCalculations
+        self.sunToCenterConstraint.constant =  diagramCalculations - sunviewCalculations - 10
```

Comment:
@alex-taffe any idea where the `10` has come from? Seems like a bit of a "Magic Number", and I'd rather avoid that if possible.